requirements/audio.txt

```diff
@@ -4,3 +4,5 @@
 # this need to be the same as used inside speechmetrics
 pesq @ git+https://github.com/ludlows/python-pesq
 pystoi <=0.3.3
+Gammatone @ https://github.com/detly/gammatone/archive/master.zip#egg=Gammatone
```

Comment:
note that this dependency will be skipped for pypi

Reply:
I tried to use `git+https://github.com/detly/gammatone`, but it failed to be installed in the test environment of `torchmetrics` as it is a conflict dependency with `speechmetrics`.

Reply:
The `speechmetrics` depends on `https://github.com/detly/gammatone/archive/master.zip#egg=Gammatone`, thus using `git+https://github.com/detly/gammatone` will report a dependency error.

Reply:
Do you have any ideas to solve it